### PR TITLE
Add simplestream maintainer docs

### PR DESCRIPTION
**Pull request by MusicDin**
Same as #35, the issue got automatically closed, probably due to visibility changes.

### CODE REVIEW

**Comment by tomponline** [comment]
Some doc failures by the looks of it (ignoring the link issue)

File: README.md [hunk 1]
```diff
@@ -1,75 +1,42 @@
...
-Use "lxd-imagebuilder [command] --help" for more information about a command.
-
-```
-<!-- Include end CLI -->
 
 <!-- Include start installing -->
 ## Installing from package
 
-`lxd-imagebuilder` is available from the [Snap Store](https://snapcraft.io/lxd-imagebuilder).
+`lxd-imagebuilder` and `simplestream-maintainer` are available from the [Snap Store](https://snapcraft.io/lxd-imagebuilder).
```
Comment (ru-fu) [inline README.md:16]: Are both tools included in the same package?

Comment (ru-fu) [reply README.md:16]: If so, let's make that clearer:

```suggestion
The `lxd-imagebuilder` and `simplestream-maintainer` tools are available in the `lxd-imagebuilder` snap from the [Snap Store](https://snapcraft.io/lxd-imagebuilder).
```

File: doc/.custom_wordlist.txt [hunk 2]
```diff
@@ -259,6 +262,8 @@ VPD
 VPS
 vSwitch
 VXLAN
+webpage
+Webpage
```
Comment (ru-fu) [inline doc/.custom_wordlist.txt:266]: This one shouldn't be needed (since it's covered by "webpage").

File: doc/howto/simps-prune.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+The `--retain-days` flag sets the maximum age of the product version and ensures that no product
+version older than the specified number of days remains on the system or product catalog.
+By default, this flag is set to `0` which means the product versions are not pruned by age.
+
+## Dangling images
+
+When pruning product versions, stream's contents are retrieved from the product catalog. This means
+there might exist an invalid product version that was not included in the product catalog (for
+example, due to the missing metadata file or mismatched checksums). By default, such versions are
+not removed.
```
Comment (ru-fu) [inline doc/howto/simps-prune.md:35]:
```suggestion
When pruning product versions, the stream's contents are retrieved from the product catalog. This means
there might exist an invalid product version that was not included in the product catalog (for
example, due to a missing metadata file or mismatched checksums). By default, such versions are
not removed.
```

**Review by tomponline — APPROVED** [review]
Thanks!

File: doc/reference/lxd-imagebuilder/index.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Reference
```
Comment (ru-fu) [inline doc/reference/lxd-imagebuilder/index.md:1]:
```suggestion
# `lxd-imagebuilder`
```

File: doc/reference/simplestream-maintainer/index.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Reference
```
Comment (ru-fu) [inline doc/reference/simplestream-maintainer/index.md:1]:
```suggestion
# `simplestream-maintainer`
```

File: doc/reference/simplestream-maintainer/directory_structure.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
+# Simple streams directory structure
+
+`simplestream-maintainer` is a CLI tool for building simple streams index and product catalog,
```
Comment (ru-fu) [inline doc/reference/simplestream-maintainer/directory_structure.md:3]:
```suggestion
`simplestream-maintainer` is a CLI tool for building a simple streams index and product catalog,
```

File: doc/reference/simplestream-maintainer/directory_structure.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+and removing expired or invalid product versions.
+
+## Terminology
+
+- `Stream`: Represents a directory that contains image builds.
+- `Product`: Represents a unique image within a single stream. It's ID is determined by the
```
Comment (ru-fu) [inline doc/reference/simplestream-maintainer/directory_structure.md:9]:
```suggestion
- `Product`: Represents a unique image within a single stream. Its ID is determined by the
```

File: doc/tutorials/imageserver.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+it has to calculate missing hashes and generate missing delta files. In such cases, we recommend
+using systemd units to prevent triggering unnecessary builds if the previous build has not finished
+yet.
+
+## Example using systemd units
+
+First create a systemd service file `/etc/systemd/system/simplestream-maintainer.service` which
+runs `simplestream-maintainer` `build` and `prune` commands as in the provided example below.
+Ensure `<simplestream_dir>` is replaced with an actual simple streams root directory and
+`<simplestream_user>` with a user that has write permissions to that directory.
```
Comment (ru-fu) [inline doc/tutorials/imageserver.md:20]: Nice! That's exactly what I meant. :+1:

File: doc/tutorials/index.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
+# Tutorials
+
+These tutorials guide you through the usage of `lxd-imagebuilder`.
```
Comment (ru-fu) [inline doc/tutorials/index.md:3]:
```suggestion
These tutorials guide you through the usage of `lxd-imagebuilder` and `simplestream-maintainer`.
```

**Review by ru-fu — COMMENTED** [review]
Late to the party I guess, but a few more small comments anyway. ;-) 
Looks great overall!

**Comment by MusicDin** [comment]
Thanks @ru-fu, will open a new PR with the suggested fixes

**Pull request merged**